Gemfile.lock

```diff
@@ -803,6 +805,7 @@ DEPENDENCIES
   uglifier (>= 1.0.3)
   unicorn-rails
   unicorn-worker-killer
+  valid_email2
```

Comment:
This could have been in the same commit as adding the gem. It's always good when each commit leaves the app in a consistent state.

Reply:
Should I do a rebase, or is it enough to live it as it is for now?

Reply:
I vote for a rebase, that can solve:
 - Having in the same commit
 - Remove the view_component upgrade from this commit/PR

Reply:
I agree, we need to remove the view_component change here. As it is this is still downgrading us to 2.34.0, I think it's because that change has already been committed. @apricot12 I would maybe try running bundle update view_component, verifying that Gemfile.lock has it at 2.35.0, then committing that result, squashing it onto that previous commit, and doing a force push. If you want to find me on Slack I can help too :)

Reply:
Yes! Looks good.